pixeltable/store.py

```diff
@@ -116,7 +116,10 @@ def create_sa_tbl(self, tbl_version: catalog.TableVersion | None = None) -> None
         # we only capture indices visible in this version
         for idx_info in tbl_version.idxs.values():
             idx = idx_info.idx.sa_index(tbl_version._store_idx_name(idx_info.id), idx_info.val_col)
-            idxs.append(idx)
+            if idx is not None:
```

Comment:
We need to leave out the indices that correspond to tbl_version.idxs, so let's remove this loop.

We still need the system indices.

Reply:
Done.